<?xml version="1.0" ?>
<launch>
  <arg name="site_name" />
  <arg name="use_sim" />

  <arg name="config_file" value="$(find rasberry_coordination)/config/map_config_$(arg site_name).yaml" unless="$(arg use_sim)"/>
  <arg name="config_file" value="$(find rasberry_coordination)/config/map_config_$(arg site_name)_sim.yaml" if="$(arg use_sim)"/>

  <node pkg="rasberry_coordination" type="visualise_agent_markers.py" name="visualise_agent_markers" args="$(arg config_file)" />

</launch>
